masterlist.yaml

```
@@ -10052,11 +10145,14 @@ plugins:
 
   - name: 'SkyrimsUniqueTreasures.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5261/' ]
+    msg:
+      - <<: *patch3rdParty_LotD
+        condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_SUT_Patch.esp")'
     clean:
-      # version: 4.4
+     # v4.4
       - crc: 0x218B87C0
         util: 'SSEEdit v4.0.3'
-
+    
   - name: 'Skyshards.esp'
```

Adding back in newline, review changes don't allow for multiline changes.
```suggestion

  - name: 'Skyshards.esp'
```